src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -78,4 +80,18 @@ public boolean delete(boolean deleteAuth0User) {
 
         return Persistence.otpUsers.removeById(this.id);
     }
+
+    /**
+     * Confirm that the requesting user has the required permissions
+     */
+    @Override
+    public boolean canBeManagedBy(RequestingUser requestingUser) {
+        if (requestingUser.apiUser.id.equals(applicationId)) {
```

Comment:
If the `requestingUser` is not an `ApiUser`, this line causes a `NullPointerException`.

Reply:
Updated.